RickAndMorty/Extensions/DateFormatter+Ext.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  dateFormatter+Ext.swift
+//  RickAndMorty
+//
+//  Created by Juan Camilo Fonseca Gomez on 13/07/23.
+//
+
+import Foundation
+
+extension DateFormatter {
+    static var iso8601Format: DateFormatter {
+        let formatter = DateFormatter()
+        formatter.locale = Locale(identifier: "en_US_POSIX")
```

Comment:
Why are you adding this default `Locale` ? What happens if you remove this?

Reply:
I remove it